Backoffice/LeftPanel/Strategies/AdministrationPanelStrategy.php

```diff
@@ -79,4 +80,24 @@ public function getRole()
     {
         return $this->role;
     }
+
+    /**
+     * @return string
+     */
+    protected function getTemplate()
+    {
+        if ($this->template) {
```

Comment:
Can you check if the template exists before using it ? It will prevent crash from typo in service configuration and enhance stability :)